<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabMeasurementObject</class>
 <widget class="QWidget" name="TabMeasurementObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>952</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>30</y>
     <width>420</width>
     <height>355</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" colspan="2">
     <widget class="QComboBox" name="comboSettings">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Settings of the detector. 
 #settings#</string>
      </property>
      <item>
       <property name="text">
        <string>Standard</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fast</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>High Gain</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dynamic Gain</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Low Gain</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Medium Gain</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Very High Gain</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Undefined</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Uninitialized</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Settings:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Number of measurements:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Acquisition Time:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="spinExpTime">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Exposure time of each frame. 
 #exptime#</string>
      </property>
      <property name="maximum">
       <double>2000000000.000000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="comboExpUnit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <item>
       <property name="text">
        <string>hr</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>min</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ms</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>us</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ns</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="spinPeriod">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Frame period between exposures. 
 #period#</string>
      </property>
      <property name="maximum">
       <double>2000000000.000000000000000</double>
      </property>
      <property name="value">
       <double>2.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="comboPeriodUnit">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <item>
       <property name="text">
        <string>hr</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>min</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ms</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>us</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ns</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Frame Period:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>File Name:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLineEdit" name="dispFileName">
      <property name="toolTip">
       <string>Root of the file name  - please check that the output directory is correctly set and select the file name format. 
 #fname#</string>
      </property>
      <property name="text">
       <string>run</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Run Index:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QSpinBox" name="spinIndex">
      <property name="toolTip">
       <string>Run index (automatically incremented) 
 #index#</string>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="suffix">
       <string/>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>2000000000</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QPushButton" name="btnStartStop">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="toolTip">
       <string>Starts Acquisition</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QSpinBox" name="spinNumMeasurements">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Number of measurements (not in real time) that will be acquired. 
 #frames#</string>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="suffix">
       <string/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>2000000000</number>
      </property>
      <property name="value">
       <number>2000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
